long-term-storage-aws.md

```diff
@@ -61,16 +78,63 @@ Once that annotation has been created and set, you'll need to attach it to the P
 For prometheus, set .Values.prometheus.serviceAccounts.server.create to false, and .Values.prometheus.serviceAccounts.server.name to the name of your created service account
 For thanos set `.Values.thanos.compact.serviceAccount`, and `.Values.thanos.store.serviceAccount` to the name of your created service account.
 
-
-
-__Thanos Encryption With S3 and KMS__
+## Thanos Encryption With S3 and KMS
 
 You can encrypt the S3 bucket where Kubecost data is stored in AWS via S3 and KMS. However, because Thanos can store potentially millions of objects, it is suggested that you use bucket-level encryption instead of object-level encryption. More details available here:
 
 https://thanos.io/tip/thanos/storage.md/#s3
```

Comment:
ZD Guide doesn't automatically make URLs clickable links. They need to follow standard markdown syntax.
I would consider making this a bulleted list as well to prevent it from being one long line of links.